src/App.jsx

```diff
@@ -1,44 +1,43 @@
-import React from 'react';
+import { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [title, setTitle] = useState(tabs[0].title);
+  const [content, setContent] = useState(tabs[0].content);
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">Selected tab is {title}</h1>
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            <Tabs
+              tabs={tabs}
+              activeTabId={activeTabId}
+              onClick={(newTitle, newContent, newActiveTabId) => {
+                setTitle(newTitle);
+                setContent(newContent);
+                setActiveTabId(newActiveTabId);
+              }}
```

Comment:
The `Tabs` component expects an `onTabSelected` prop, but here you are passing an `onClick` prop. You should rename `onClick` to `onTabSelected` to match the expected prop name in the `Tabs` component.